Documentation/QRCodeSign.md

```diff
@@ -0,0 +1,28 @@
+**Instructions**
+
+ All flowers are labeled, allowing visitors to take note of top performing cultivars under regional conditions.
+ 
+ <br>
+ 
+**Search for your favorite flower**
+
+ There are three ways to get to the flower you are looking for:
```

Comment:
I guess I could say
"There are three ways to get to the flower page you are interested in" ?

What we tried to say is that you can either scan the QR code, search it in the search bar, or use the dropdown menu for the bed